src/types/validate-slack-request/index.d.ts

```diff
@@ -0,0 +1,7 @@
+import { Request } from "express";
+
+export default function validateSlackRequest(
```

Comment:
We can define types in third-party modules like this instead:
```ts
declare module "validate-slack-request" {
    // type declarations...
}
```

Also, is there any official library that can check request signature? Does `@slack/events-api` have such functions? I'd rather use an official library than a library with little usage in wild.